vividus-extension-selenium/src/main/java/org/vividus/ui/screenshot/AbstractScreenshotParametersFactory.java

```diff
@@ -118,8 +118,10 @@ private P createParameters(C configuration, Map<IgnoreStrategy, Set<Locator>> st
     {
         P parameters = createScreenshotParameters();
         parameters.setShootingStrategy(configuration.getShootingStrategy());
-        parameters.setNativeFooterToCut(ensureValidCutSize(configuration.getNativeFooterToCut(), "native footer"));
-
+        parameters.setCutTop(configuration.getCutTop());
```

Comment:
ensureValidCutSize ? :)

Reply:
I was thinking about that, but I am not really sure if it makes any sense. Cut top -10 pixels.
We don't really do any validation if the provided cuts are not overflow screenshot size

Reply:
Also was thinking about this, we can set cut 10000 to get image out of bound errors, but I think these are different types of errors, -10 pixels is logically invalid value since cut size can not be negative value, whereas 10000 is logically valid